Develop

vic778 opened the pull request:
In this activity, 
- I set up a JavaScript project for the Leaderboard list application, using webpack and ES6 features, i
- including modules. I developed the first functional version of the application following a thread,

vic778 committed:
Added linter folder for github

vic778 committed:
Added dist folder for deplyment

vic778 committed:
Added screen

vic778 committed:
Created data

vic778 committed:
Created file for linter

vic778 committed:
Added wepack plugin

vic778 committed:
Added  screenshot

vic778 committed:
FIxed linter errors

vic778 committed:
Added plugin

vic778 committed:
Removed .js

vic778 committed:
Added build folder

vic778 committed:
Added plugin

vic778 committed:
Ignore

vic778 committed:
Afixed linter error

vic778 committed:
Fixed linter error

vic778 committed:
Added screen

vic778 committed:
Added live demo link

vic778 committed:
Update README.md

vic778 committed:
Update package.json

Added the plugin

vic778 committed:
Update README.md

dasileker reviewed (approved):
## **Hi @vic778,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

vic778 committed:
Created function to send data

vic778 committed:
Added padding alll body

vic778 committed:
Added screenshot

vic778 committed:
Created all methods to fetchinh with data

vic778 committed:
Added stylesheet

vic778 committed:
Update README.md

vic778 committed:
Update README.md

vic778 committed:
Merge pull request #3 from vic778/style-project

Style project

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 277.51kb -> 230.02kb (17.11%)

/img/screen2.png -- 152.93kb -> 126.42kb (17.33%)
/img/screen1.png -- 124.59kb -> 103.60kb (16.84%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

vic778 committed:
Merge pull request #4 from vic778/imgbot

[ImgBot] Optimize images
